plugins/koa/koaintegrationplugin.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const compose = require('koa-compose');
+const Router = require('koa-router');
+const router = new Router();
+const reconfig = require('@brillout/reconfig');
+
+module.exports = {
```

Comment:
Let's exports the routes directly. I guess we can rename the file to koaIntegrationRoutes.js.

syntax: camel case filenames is becoming a standard in JS world

Reply:
Ah, actually, it's the function that should be renamed to `middleware`. So I guess we could rename the file to `koaIntegrationMiddleware.js`

Reply:
Done